models/ios/vxlan_vtep/ios_vxlan_vtep.yaml

```diff
@@ -0,0 +1,97 @@
+module: ios_vxlan_vtep
+short_description: Resource module to configure VXLAN VTEP interface.
+description: This module provides declarative management of VXLAN VTEP interface on Cisco IOS network
+  devices.
+version_added: 1.0.0
+author: Padmini Priyadarshini Sivaraj
+notes:
+  - Tested against Cisco IOSl2 device with Version 15.2 on VIRL.
```

Comment:
the version of cisco ios xe appliance is 17+ I suppose

Reply:
we are testing on cat9k on CML not VIRL

Reply:
Corrected